utils/node_local_test.py

```diff
@@ -3898,6 +3898,11 @@ def test_pil4dfs(self):
         self.server.run_daos_client_cmd_pil4dfs(['ln', '-s', file4, link1])
         self.server.run_daos_client_cmd_pil4dfs(['rm', '-Rf', dir1])
 
+        # dd to write a file
+        file5 = join(path, 'newfile')
+        self.server.run_daos_client_cmd_pil4dfs(['dd', 'if=/dev/zero', f'of={file5}', 'bs=1',
+                                                'count=1024'])
+
```

Comment:
This will run the test twice, once with dfuse caching and once without but that should be OK.  If needs be there is an option to just run without caching that some methods use but I can't remember the details.

Reply:
Good to know. Thank you!